tests/sqllogictests/suites/base/03_common/03_0040_parquet_small_files

```diff
@@ -0,0 +1,45 @@
+statement ok
+drop stage if exists r2_stage;
+
+statement ok
+drop table if exists contract;
+
+statement ok
+CREATE TABLE `contract` ( `block_time` BIGINT NULL, `address` VARCHAR NULL, `event_signature` VARCHAR NULL, `event_params` ARRAY(STRING NULL) NULL, `block_number` BIGINT NULL, `block_hash` VARCHAR NULL, `log_index` BIGINT NULL, `transaction_hash` VARCHAR NULL, `transaction_index` BIGINT NULL, `data` VARCHAR NULL, `topics` VARCHAR NULL, `id` VARCHAR NULL);
+
+statement ok
+CREATE STAGE r2_stage URL='s3://indexed-xyz/ethereum/decoded/logs/v1.2.0/partition_key=9d/dt=2016/' CONNECTION = ( ENDPOINT_URL = 'https://ed5d915e0259fcddb2ab1ce5592040c3.r2.cloudflarestorage.com/' REGION = 'auto' ACCESS_KEY_ID = '43c31ff797ec2387177cabab6d18f15a' SECRET_ACCESS_KEY = 'afb354f05026f2512557922974e9dd2fdb21e5c2f5cbf929b35f0645fb284cf7' );
```

Comment:
Unfortunately, this R2 data set is not mutable, the data is updated sometimes

Reply:
I see, I will try to gen some parquet files for tests.